README.md

```diff
@@ -15,20 +15,69 @@ $ pytest --cov=.
 ```
 
 ### Deployment
+=======
 
-We use [ansible](https://www.ansible.com/) to deploy our apps and configure servers, which adds a more dependencies:
+To deploy, you need to do a few things:
 
-```bash
-# This is intentionally excluded from the requirements.txt, and should
-# be done outside a virtualenv (ansible doesn't work properly with virtualenvs)
-$ pip3 install ansible
-```
+1. Install Docker and Docker Compose by following the instructions on the [Docker website](https://docs.docker.com/compose/install/)
 
-To deploy, you need to do a few things:
+2. Run Docker Compose in detached mode `-d`:
```

Comment:
This section moved to [CONTRIBUTING.md](https://github.com/pyslackers/website/blob/master/CONTRIBUTING.md)